recipe/build.sh

```diff
@@ -88,3 +88,9 @@ rm -f ./temp_prefix/lib/libboost_python*
 rm -f ./temp_prefix/lib/libboost_numpy*
 rm -rf ./temp_prefix/lib/cmake/boost_python*
 rm -rf ./temp_prefix/lib/cmake/boost_numpy*
+
+# Use a larger default for pre-generated headers.
+# This generates more macros for larger sizes.
+# See include/boost/hana/detail/struct_macros.hpp.erb for more details
+export MAX_NUMBER_OF_MEMBERS=200
+erb include/boost/hana/detail/struct_macros.hpp.erb > temp_prefix/include/boost/hana/detail/struct_macros.hpp
```

Comment:
I think this might be
```suggestion
erb libs/hana/include/boost/hana/detail/struct_macros.hpp.erb > temp_prefix/include/boost/hana/detail/struct_macros.hpp
```